build.sbt

```diff
@@ -184,7 +183,7 @@ ThisBuild / coreDependencies := (providedSparkDependencies.value ++ testCoreDepe
   "com.github.broadinstitute" % "picard" % "2.27.5",
   "org.apache.commons" % "commons-lang3" % "3.14.0",
   // Fix versions of libraries that are depended on multiple times
-  "org.apache.hadoop" % "hadoop-client" % "3.4.0",
+  "org.apache.hadoop" % "hadoop-client" % "3.3.6",
```

Comment:
For some reason, version 3.4.0 broke the assembly jar (couldn't load Glow classes). I added a test to make sure this can't happen again.